admin/workspace-providers/management.md

```diff
@@ -60,16 +60,49 @@ At this point, you can:
   > If you enable **end-to-end encryption**, end-users using SSH need to rerun
   > `coder config-ssh`.
 
-- Specify the Kubernetes `tolerations` and `nodeSelector` for the workspaces
-  deployed with this provider:
+- Specify the Kubernetes `pod_tolerations`, `pod_node_selector`, and
```

Comment:
@f0ssel do you mind reviewing these changes re: affinities?

Reply:
I added some updates in ed7e880, d030118, and a457195.